### PR TITLE
Added validation for assistance info in detail view

### PR DESCRIPTION
**Pull request by luisalduucin**
Fixes issue #80

### CODE REVIEW

File: grails-app/views/event/detail.gsp [hunk 1]
```diff
@@ -152,12 +162,19 @@
...
+                                                        <g:if test="${enrollment.attendance}">
+                                                            <span class="label label-primary"><i class="fa fa-check"></i> Assisted</span>
+                                                        </g:if>
+                                                        <g:else>
+                                                            <span class="label label"><i class="fa fa-close"></i> Not Assisted</span>
+                                                        </g:else>
                                                     </g:if>
                                                     <g:else>
-                                                        <span class="label label"><i class="fa fa-close"></i> Not Assisted</span>
+                                                        <span class="label label"><i class="fa fa-calendar"></i> Pending</span>
```
Comment (blzb) [inline grails-app/views/event/detail.gsp]: Maybe not show the label if it's pending

File: grails-app/views/event/detail.gsp [hunk 2]
```diff
@@ -152,12 +162,19 @@
                                                 <td style="vertical-align:middle"><strong>${enrollment.attendee.name}</strong></td>
                                                 <td style="vertical-align:middle"><strong>${enrollment.attendee.school}</strong></td>
```
Comment (blzb) [inline grails-app/views/event/detail.gsp]: Change for email instead

File: grails-app/views/event/detail.gsp [hunk 3]
```diff
@@ -152,12 +162,19 @@
...
                                                 <td style="vertical-align:middle">
-                                                    <g:if test="${enrollment.attendance}">
-                                                        <span class="label label-primary"><i class="fa fa-check"></i> Assisted</span>
+
+                                                    <g:if test="${eventDetails.expired}">
+                                                        <g:if test="${enrollment.attendance}">
+                                                            <span class="label label-primary"><i class="fa fa-check"></i> Assisted</span>
+                                                        </g:if>
+                                                        <g:else>
+                                                            <span class="label label"><i class="fa fa-close"></i> Not Assisted</span>
```
Comment (blzb) [inline grails-app/views/event/detail.gsp]: label class repeated

**Comment by luisalduucin** [comment]
Change the active label of the event description when expired

**Review by imarban — APPROVED** [review]

**Pull request merged**